agent/api/transitiondependency.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+// TransitionDependencySet contains dependencies that impact transitions of
+// containers.
+type TransitionDependencySet struct {
+	// ContainerDependencies is the set of containers on which a transition is
+	// dependent.
+	ContainerDependencies []ContainerDependency `json:"ContainerDependencies"`
+}
+
+// ContainerDependency defines the relationship between a dependent container
+// and its dependency.
+type ContainerDependency struct {
+	// Container defines the container on which a transition depends
+	Container string `json:"Container"`
```

Comment:
Would `ContainerName` be more accurate here, as `Container` is widely used as a container struct?

Reply:
Done